content/news/2023-10-13-EGD-2023-the-wrapup/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: The European Galaxy Days are over...
+date: '2023-10-13'
+tease: "...but we look back with a relaxed smile. Let's draw the balance!"
+location:
+  name: Freiburg, Germany
+tags: [meeting, European Galaxy Days, community, ELIXIR, EuroScienceGateway]
+authors: Sebastian Schaaf, Anika Erxleben-Eggenhofer, Björn Grüning, Hans-Rudolf Hotz
+authors_structured:
+- github: sebastian-schaaf
+- github: erxleben
+- github: bgruening
+- github: hrhotz
+supporters:
+- ELIXIR
+- denbi
+components: true
+autotoc: true
+subsites: [all, esg]
+---
+
+<div class="float-right" style="max-width: 400px">
+
+![logo EGD](logo-egd2023.png)
+
+</div>
+
+About a week ago the [2023 edition of the European Galaxy Days (EGD 2023)](/events/2023-10-egd) has ended, and we can say: it was a pleasure to have you as guests in Freiburg, including those who participated virtually. The three days were as intense as exciting, no matter whether during the talks, between the session at coffee or lunch, or in the (partly long) evenings in various gastronomical outposts in the city.
```

Comment:
do we write it for the participants? Otherwise I would not write "it was a pleasure to have you as guests in Freiburg"